demo/app.component.html

```diff
@@ -229,6 +229,23 @@
         [roundDomains]="roundDomains"
         (select)="select($event)">
       </ngx-charts-line-chart>
+      <ngx-charts-bubble-chart
+        *ngIf="chartType === 'bubble-chart'"
+        [view]="view"
+        class="chart-container"
+        [results]="bubble"
+        [showGridLines]="showGridLines"
+        [legend]="showLegend"
+        [xAxis]="showXAxis"
+        [yAxis]="showYAxis"
+        [showXAxisLabel]="showXAxisLabel"
+        [showYAxisLabel]="showYAxisLabel"
+        [xAxisLabel]="xAxisLabel"
+        [yAxisLabel]="yAxisLabel"
+        [scheme]="colorScheme"
+        [schemeType]="schemeType"
```

Comment:
Do we want to support linear schemeType? If so, we'd probably need two of these, for both x and y. It's a cool feature to have, but probably complex to implement. See the implementation of it in line chart.